collector/perf_schema_replication_group_members.go

```diff
@@ -0,0 +1,149 @@
+// Copyright 2018 The Prometheus Authors
```

Comment:
New files should contain the year they're created.

```suggestion
// Copyright 2020 The Prometheus Authors
```